src/Components/Publishing/Display/Canvas/CanvasText.tsx

```diff
@@ -16,13 +17,18 @@ interface DivProps extends React.HTMLProps<HTMLDivElement> {
 
 export const CanvasText: React.SFC<CanvasTextProps> = props => {
   const { disclaimer, unit } = props
+  let resized_logo = resize(unit.logo, {
```

Comment:
If this variable will never change then best to use `const`. `let` is typically used to signify reassignment at some later point, while `const` will throw an error in the compiler as well as the linter if reassignment is attempted.